### PR TITLE
Add typos discovered in django/deps repository

### PR DESCRIPTION
**Pull request by jdufresne**
https://github.com/search?q=repo%3Adjango%2Fdeps+typo&type=pullrequests

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -2923,6 +2923,9 @@ approoximates->approximates
 approoximating->approximating
 approoximation->approximation
 approoximations->approximations
+appropach->approach
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:2926]: Could you add all verbal forms?
```
appropached->approached
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -3336,6 +3339,7 @@ arhive->archive
 arhives->archives
 aribitary->arbitrary
 aribiter->arbiter
+aribitrary->arbitrary
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:3342]: Perhaps also:
```
aribitrarily->arbitrarily
```

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -21521,6 +21526,7 @@ implementes->implements
 implementet->implemented
 implemention->implementation
 implementions->implementations
+implementor->implementer
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:21529]: Plural?
```
implementors->implementers
```

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -21593,7 +21599,7 @@ implimented->implemented
 implimenting->implementing
 implimention->implementation
 implimentions->implementations
-implimentor->implementor
+implimentor->implementer
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:21602]: Plural?
```
implimentors->implementers
```

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -24385,6 +24393,7 @@ leyered->layered
 leyering->layering
 leyers->layers
 liares->liars
+liase->liaise
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:24396]: All verbal forms?
```
liased->liaised
liases->liaises
liasing->liaising
```

**Review by DimitriPapadopoulos — COMMENTED** [review]
A few suggestions (plural and all verbal forms).

**Comment by jdufresne** [comment]
@DimitriPapadopoulos Thanks for the review. I applied all suggestions in "fixup! Add typos discovered in django/deps repository".

**Review by DimitriPapadopoulos — APPROVED** [review]

**Comment by DimitriPapadopoulos** [comment]
@jdufresne Thank you for your valuable contributions to codespell.

**Pull request merged**